ops/ccs-ops-misc/load_test/common/pull_bene_ids.py

```diff
@@ -0,0 +1,48 @@
+import os
+import psycopg2 as p
+import yaml
+from common import config
+
+'''
+Calls out to the database located in the config file, and returns the list of beneficiary ids from the query.
+
+Returns a list of beneficiary ids, or an empty array if nothing was found.
+'''
+def loadData():
+    ## Load configuration data, like db creds
+    configFile = config.load()
+
+    ## if we failed to load the config, bail out
+    if configFile is None:
+        return []
```

Comment:
Should we print an error message here?

Reply:
same comment for other places below

Reply:
Good call, I'll add an error message here.

Reply:
Oh wait I forgot, in config.load() if it fails to load, it will print an error message there; I left this blank because they should already know it died from down the stack.